recipe/build.sh

```diff
@@ -21,4 +21,5 @@ fi
 ./configure --with-pic --prefix=$PREFIX --disable-dependency-tracking
 make
 make check
+ln README.md README  # hack; snappy autotools build is broken in 1.1.6
```

Comment:
> This is a hard link

Right. Missed that. `conda-build` used to have issues with those, but it is pretty good about breaking them these days (by copying the file around). IOW NVM about copying.

> I guess I'll stop being lazy, fix the autotools script, put it as a patch here, and send a PR upstream, because why not fix it for everyone....

Probably a good idea too.